### PR TITLE
Add new prefix: semapv

### PR DESCRIPTION
**Pull request by cthoyt**
Closes https://github.com/mapping-commons/semantic-mapping-vocabulary/issues/5.

@matentzn can you please check that this is all correct? Is this how we should be calling this, or should it be semapv_voc?

Also somehow I thought we added this before...

### CODE REVIEW

**Comment by cthoyt** [comment]
@matentzn can I add your email address to be associated as the contact person? Do you have a preference which email to use?

File: src/bioregistry/data/bioregistry.json [hunk 1]
```diff
@@ -86649,6 +86649,30 @@
...
+      "email": "cthoyt@gmail.com",
+      "github": "cthoyt",
+      "name": "Charles Tapley Hoyt",
+      "orcid": "0000-0003-4423-4370"
+    },
+    "description": "The Semantic Mapping Vocabulary provides and defines terms used for creating and maintaining semantic mappings, in particular mapping metadata.",
+    "download_owl": "https://raw.githubusercontent.com/mapping-commons/semantic-mapping-vocabulary/main/semapv.owl",
+    "example": "BackgroundKnowledgeBasedMatching",
+    "homepage": "https://mapping-commons.github.io/semantic-mapping-vocabulary/",
+    "license": "CC-BY-4.0",
```
Comment (matentzn) [inline src/bioregistry/data/bioregistry.json:86669]:
```suggestion
    "license": "CC-0",
```

Comment (matentzn) [reply src/bioregistry/data/bioregistry.json:86669]: https://github.com/mapping-commons/semantic-mapping-vocabulary/pull/16

File: src/bioregistry/data/bioregistry.json [hunk 2]
```diff
@@ -86649,6 +86649,30 @@
...
+    },
+    "contributor": {
+      "email": "cthoyt@gmail.com",
+      "github": "cthoyt",
+      "name": "Charles Tapley Hoyt",
+      "orcid": "0000-0003-4423-4370"
+    },
+    "description": "The Semantic Mapping Vocabulary provides and defines terms used for creating and maintaining semantic mappings, in particular mapping metadata.",
+    "download_owl": "https://raw.githubusercontent.com/mapping-commons/semantic-mapping-vocabulary/main/semapv.owl",
+    "example": "BackgroundKnowledgeBasedMatching",
```
[the remark this answers is not in the document]
Comment (matentzn) [reply src/bioregistry/data/bioregistry.json:86667]: Easier to comprehend example.

**Review by matentzn — COMMENTED** [review]
Two small fixes, otherwise great.

File: src/bioregistry/data/bioregistry.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (cthoyt) [reply src/bioregistry/data/bioregistry.json:86667]: we can add multiple examples if you prefer!

File: src/bioregistry/data/bioregistry.json (hunk 1 above)
Comment (cthoyt) [reply src/bioregistry/data/bioregistry.json:86669]: hella

File: src/bioregistry/data/bioregistry.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (matentzn) [reply src/bioregistry/data/bioregistry.json:86667]: I think its fine, multiple examples also mean more maintenance.. I think the linked documentation is the most important!

**Pull request merged**